core/modules/search/templates/_results_normal.inc.php

```diff
@@ -148,7 +148,7 @@
                         <?php echo (!$issue->hasEstimatedTime()) ? '-' : \thebuggenie\core\entities\Issue::getFormattedTime($issue->getEstimatedTime(true, true)); ?>
                     </td>
                     <td class="sc_spent_time<?php if (!$issue->hasSpentTime()): ?> faded_out<?php endif; ?>"<?php if (!in_array('spent_time', $visible_columns)): ?> style="display: none;"<?php endif; ?>>
-                        <?php echo (!$issue->hasSpentTime()) ? '-' : \thebuggenie\core\entities\Issue::getFormattedTime($issue->getSpentTime(true, true)); ?>
+                        <?php echo (!$issue->hasSpentTime() || !$issue->isSpentTimeVisibleByPermission()) ? '-' : \thebuggenie\core\entities\Issue::getFormattedTime($issue->getSpentTime(true, true)); ?>
```

Comment:
Yes, it's by design. The reason it does this is because if an issue has the value set, even when the issue scheme doesn't allow it, then it's probably important enough to be visible. I'm not sure we need the actual distinction, since most of the time this won't occur. Opinions?